cruise.umple/test/cruise/umple/compiler/487_parallelStateMachines_sameRegionDiffNames.ump

```diff
@@ -0,0 +1,25 @@
+class X {
```

Comment:
In my tests --> assertNoWarnings("487_parallelStateMachines_sameRegionDiffNames.ump");